GitOut/Features/Git/Log/Converters/TreeToMarginConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Windows;
+using System.Windows.Data;
+
+namespace GitOut.Features.Git.Log.Converters
+{
+    public class TreeToMarginConverter : IValueConverter
+    {
+        public const int Distance = 20;
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (!(value is GitTreeEvent gitTreeEvent))
+            {
+                return DependencyProperty.UnsetValue;
+            }
+
+            int maxIndex = gitTreeEvent.Nodes
+                .Select(n => new [] { n.Top?.Up ?? 0, n.Top?.Down ?? 0, n.Bottom?.Down ?? 0})
```

Comment:
Actually it seems as though this is incorrectly formatted, my VS formats `new[] {` (no space between new and [])